utils/utils.go

```diff
@@ -40,6 +42,8 @@ func RemovePolishCharacters(str string) (string, error) {
 				return 'z'
 			case 'ź':
 				return 'z'
+			case '?':
```

Comment:
thats not the best idea i think, we could just remove those characters instead replacing with whitespace.

Reply:
ok